demo/server/src/modules/index.ts

```diff
@@ -4,10 +4,13 @@ import * as express from 'express';
 import { connect as syncConnector } from './datasync/Connector';
 import { router as syncRouter } from './datasync/Router';
 import { init as authInit } from './passport-auth';
+import { init as initKeycloak } from './keycloak';
 
 export let securityMiddleware: EndpointSecurity;
 export let logger: Logger;
 
+const useKeycloak = true;
```

Comment:
@TommyJ1994 so this means that keycloak.json we adding it's not going to be in the final demo app? Best to always have keycloak.json in the demo and just additional configuration that will point to keycloak server.